src/org/openj9/envInfo/JavaInfo.java

```diff
@@ -102,7 +104,7 @@ public String getJDKImpl() {
         impl = impl.toLowerCase();
         if (impl.contains("ibm")) {
             return "ibm";
-        } else if (impl.contains("openj9")) {
+        } else if (impl.contains("openj9") || impl.contains("OpenJ9")) {
```

Comment:
`impl` is set to toLowerCase() at line 104. I do not think this change is needed.

Reply:
I was seeing this message before I put the change in:
```
Cannot determine System.getProperty('os.arch')=riscv

System.getProperty('java.vm.name')=Eclipse OpenJ9 VM
```

Reply:
Please revert the above change and try again.

Reply:
Ah, sorry, I've just seen it prints it before it's set to `toLowerCase()` and that'll be what I'm seeing.